src/navigation/vertical/index.js

```diff
@@ -9,68 +9,60 @@ import CreditCardOutline from 'mdi-material-ui/CreditCardOutline'
 import AccountPlusOutline from 'mdi-material-ui/AccountPlusOutline'
 import AlertCircleOutline from 'mdi-material-ui/AlertCircleOutline'
 import GoogleCirclesExtended from 'mdi-material-ui/GoogleCirclesExtended'
+import { useSession } from 'next-auth/react'
 
 const navigation = () => {
+  const {status, data } = useSession();
+  console.log(data);
+
+  if(status === "authenticated")
+    return [
+      {
+        title: 'Dashboard',
+        icon: HomeOutline,
+        path: '/'
+      },
+      {
+        title: 'Account Settings',
```

Comment:
passing by getServerSideProps will let us to not even enter the pages so verifying the navbar will never be important in this case

Reply:
ok I'll search about that